packages/api/src/schema/datasets/gnomad_r2_0_2/mergeExomeAndGenomeVariants.js

```diff
@@ -0,0 +1,106 @@
+const add = (n1, n2) => (n1 || 0) + (n2 || 0)
+
+const mergeExomeAndGenomeVariants = (exomeVariants, genomeVariants) => {
```

Comment:
We may need to think of a more general way to merge variant datasets down the road -- it will be a requirement for other projects as well.